package.json

```diff
@@ -37,7 +37,7 @@
     "gatsby-source-git": "^1.0.2",
     "gatsby-transformer-remark": "^2.6.32",
     "html-react-parser": "^0.9.2",
-    "node-sass": "^7.0.0",
+    "node-sass": "^4.13.0",
```

Comment:
It's reverting the change that [dependabot made](https://github.com/silverstripe/doc.silverstripe.org/pull/236). We need to revert this otherwise it won't deploy.